.github/workflows/build-and-test.yml

```diff
@@ -77,8 +79,32 @@ jobs:
         run: npm ci
 
       - name: Tag repo & create releases
+        id: changesets_publish
         uses: changesets/action@v1.4.5
         with:
           publish: npm run ci:publish
         env:
-          GITHUB_TOKEN: ${{ secrets.CHANGESETS_GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Parse published version
+        id: parse_version
+        uses: madhead/semver-utils@v3.1.0
+        with:
+          version: ${{ steps.changesets_publish.outputs.publishedPackages[0].version }}
```

Comment:
If we published a new version of the action (the `published` output is true) then there will always be 1 entry in `publishedPackages` which will contain the new version.